avm/res/key-vault/vault/key/main.bicep

```diff
@@ -150,6 +150,12 @@ resource key_roleAssignments 'Microsoft.Authorization/roleAssignments@2022-04-01
   }
 ]
 
+@description('The uri of the key.')
```

Comment:
Hey @fblix sorry to chime in on one point, then I'll leave the PR review to you.

@Gosselin-l please also update the minor version in the version.json file, since a new output is added functionality for the module. Thanks!